### PR TITLE
docs(jfrog): add bonita artifact repository installation

### PR DESCRIPTION
**Pull request by akantcheff**
BTT is now available on new Bonita Artifact Repository.

This PR is linked to https://github.com/bonitasoft/bonita-doc/pull/2079 

JIRA [CI-714](https://bonitasoft.atlassian.net/browse/CI-714)

### CODE REVIEW

**Comment by rbioteau** [comment]
Depends on https://github.com/bonitasoft/bonita-doc/pull/2079/files

**Comment by uguy** [comment]
Waiting for PR https://github.com/bonitasoft/bonita-doc/pull/2079 to be merged

File: modules/ROOT/pages/installation.adoc [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
 Integrations tests are built upon the Bonita Test Toolkit, based on the open source https://github.com/bonitasoft/bonita-java-client[Bonita Java Client]. +
-The Bonita Test Toolkit is only available for Enterprise, Performance, Efficiency, and Teamwork. editions. 
+The Bonita Test Toolkit is only available for Enterprise, Performance, Efficiency, and Teamwork. editions.
 ====
 
+The `bonita-test-toolkit` can be installed in three different ways. Choose the one which suits you best.
+
+Once installed, you can start using the `bonita-test-toolkit` dependency from any Maven project. Have a look at the xref:quick-start.adoc[quick start guide] for a first hands-on.
+
+== Bonita Artifact Repository installation
```
Comment (ghost) [inline modules/ROOT/pages/installation.adoc]: I suggest to rephrase to "**Installation from** the Bonita Artifact Repository"

File: modules/ROOT/pages/installation.adoc [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
+
+The Bonita Test Toolkit is available on xref:bonita:software-extensibility:bonita-repository-access.adoc#bonita-artifact-repository[Bonita Artifact Repository] as a standard Maven dependency.
+Read the xref:bonita:software-extensibility:bonita-repository-access.adoc#maven[Maven section] to see how to configure your Maven settings in order to access to Bonita Artifact Repository.
+
 == Local repository installation
```
Comment (ghost) [inline modules/ROOT/pages/installation.adoc]: I suggest to rephrase to "**Installation from the Customer Portal**"

File: modules/ROOT/pages/installation.adoc [hunk 3]
```diff
@@ -1,22 +1,29 @@
...
+The Bonita Test Toolkit dependency can also be downloaded from the https://customer.bonitasoft.com/download/request[Bonitasoft Customer Portal].
 
 * Choose the Bonita-Subscription-2022.1-uX product and click on `Access Download Page`.
 * Scroll down to the Maven repository archive and download it.
-* Extract its content in your home folder in: `$USER_HOME/.m2/repository`. This is the default location of a Maven local repository. If you are already familiar with Maven and use another local repository location, extract the content there.
-
-Once installed, you can start using the `bonita-test-toolkit` dependency from a Maven project. Have a look at the xref:quick-start.adoc[quick start guide] for a first hands-on.
+* Extract its content in your home folder in: `$USER_HOME/.m2/repository`. This is the default location of a https://maven.apache.org/settings.html#simple-values[Maven local repository]. If you are already familiar with Maven and use another local repository location, extract the content there.
 
 == Bonita Continuous Delivery installation
```
Comment (ghost) [inline modules/ROOT/pages/installation.adoc]: I suggest to rephrase "**Installation from** Bonita Continuous Delivery"

**Review by uguy — CHANGES_REQUESTED** [review]
LGTM, but need to change `.github/workflows/build-pr-preview.yml` to not target PR branches in preview.

**Comment by rbioteau** [comment]
:partying_face: 
@akantcheff It was a bad module reference in the link + missing bonita version to build the preview

**Comment by akantcheff** [comment]
> 🥳 @akantcheff It was a bad module reference in the link + missing bonita version to build the preview

The preview won't work until the PR https://github.com/bonitasoft/bonita-doc/pull/2079 is merged.

**Comment by rbioteau** [comment]
https://github.com/bonitasoft/bonita-doc/pull/2079 is missing
https://github.com/bonitasoft/bonita-doc/tree/2022.2/modules/software-extensibility/pages

:thinking:

**Comment by rbioteau** [comment]
OOps I thought it was merged for some reasons....
I fixed the links so it should be good when the pr is merged.

**Comment by akantcheff** [comment]
Yes we create a new page in https://github.com/bonitasoft/bonita-doc/pull/2079 that we reference in this one so the preview is broken.

**Pull request merged**